content/academy/apify_platform/deploying/docker_file.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Dockerfile
+description: Understand how to write a Dockerfile (Docker image blueprint) for your project so that it can be run within a Docker container on the Apify platform.
+menuWeight: 3
+paths:
+- apify-platform/deploying/docker-file
+---
+
+# [](#dockerfile) Dockerfile
+
+The **Dockerfile** is a file which gives the Apify platform (or Docker, more specifically) instructions on how to create an environment for your code to run in. Every actor must have a Dockerfile, as actors run in Docker containers.
+
+> Actors on the platform are always run in Docker containers; however, they can also be run in local Docker containers. This is not common pracice though, as it requires more setup and a deeper understanding of Docker. For testing, it's best to just run the actor on the local OS (this requires you to have the underlying runtime installed, such as Node.js, Python, Rust, GO, etc).
```

Comment:
typo `pracice`

Reply:
![CleanShot 2022-04-28 at 22 09 49@2x](https://user-images.githubusercontent.com/23726914/165837246-91812950-6f00-4e7f-a0d8-0ceb1e19da27.png)
It still highlights the wrong lesson in the menu.

Reply:
Oh, super weird. Probably needs a rename then. Suggestions?

Reply:
Maybe you just need to rename the parent deploying.md file to deploying_your_code.md and it will fix it.

Reply:
Renamed